error: Could not connect to the database. Access denied for user 'SyedMazharAli'@'localhost' (using password: YES) {"name":"SequelizeAccessDeniedError","original":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: YES)","sqlState":"28000"},"parent":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: YES)","sqlState":"28000"},"stack":"SequelizeAccessDeniedError: Access denied for user 'SyedMazharAli'@'localhost' (using password: YES)\n    at ConnectionManager.connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\mysql\\connection-manager.js:118:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at ConnectionManager._connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:332:24)\n    at C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:250:32","timestamp":"2023-04-19T15:58:57.059Z"}
info: Server started at http://localhost:3005/api/v1/ {"timestamp":"2023-04-19T15:59:25.821Z"}
error: Could not connect to the database. Access denied for user 'SyedMazharAli'@'localhost' (using password: YES) {"name":"SequelizeAccessDeniedError","original":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: YES)","sqlState":"28000"},"parent":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: YES)","sqlState":"28000"},"stack":"SequelizeAccessDeniedError: Access denied for user 'SyedMazharAli'@'localhost' (using password: YES)\n    at ConnectionManager.connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\mysql\\connection-manager.js:118:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at ConnectionManager._connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:332:24)\n    at C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:250:32","timestamp":"2023-04-19T15:59:25.890Z"}
info: Server started at http://localhost:3005/api/v1/ {"timestamp":"2023-04-19T16:01:54.569Z"}
error: Could not connect to the database. Access denied for user 'SyedMazharAli'@'localhost' (using password: YES) {"name":"SequelizeAccessDeniedError","original":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: YES)","sqlState":"28000"},"parent":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: YES)","sqlState":"28000"},"stack":"SequelizeAccessDeniedError: Access denied for user 'SyedMazharAli'@'localhost' (using password: YES)\n    at ConnectionManager.connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\mysql\\connection-manager.js:118:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at ConnectionManager._connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:332:24)\n    at C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:250:32","timestamp":"2023-04-19T16:01:54.645Z"}
info: Server started at http://localhost:3005/api/v1/ {"timestamp":"2023-04-19T16:06:00.827Z"}
error: Could not connect to the database. Access denied for user 'SyedMazharAli'@'localhost' (using password: YES) {"name":"SequelizeAccessDeniedError","original":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: YES)","sqlState":"28000"},"parent":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: YES)","sqlState":"28000"},"stack":"SequelizeAccessDeniedError: Access denied for user 'SyedMazharAli'@'localhost' (using password: YES)\n    at ConnectionManager.connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\mysql\\connection-manager.js:118:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at ConnectionManager._connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:332:24)\n    at C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:250:32","timestamp":"2023-04-19T16:06:00.905Z"}
info: Server started at http://localhost:3005/api/v1/ {"timestamp":"2023-04-19T16:07:42.478Z"}
error: Could not connect to the database. Access denied for user 'SyedMazharAli'@'localhost' (using password: YES) {"name":"SequelizeAccessDeniedError","original":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: YES)","sqlState":"28000"},"parent":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: YES)","sqlState":"28000"},"stack":"SequelizeAccessDeniedError: Access denied for user 'SyedMazharAli'@'localhost' (using password: YES)\n    at ConnectionManager.connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\mysql\\connection-manager.js:118:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at ConnectionManager._connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:332:24)\n    at C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:250:32","timestamp":"2023-04-19T16:07:42.552Z"}
info: Server started at http://localhost:3005/api/v1/ {"timestamp":"2023-04-19T16:09:36.065Z"}
error: Could not connect to the database. Access denied for user 'SyedMazharAli'@'localhost' (using password: YES) {"name":"SequelizeAccessDeniedError","original":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: YES)","sqlState":"28000"},"parent":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: YES)","sqlState":"28000"},"stack":"SequelizeAccessDeniedError: Access denied for user 'SyedMazharAli'@'localhost' (using password: YES)\n    at ConnectionManager.connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\mysql\\connection-manager.js:118:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at ConnectionManager._connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:332:24)\n    at C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:250:32","timestamp":"2023-04-19T16:09:36.144Z"}
info: Server started at http://localhost:3005/api/v1/ {"timestamp":"2023-04-19T16:13:24.703Z"}
error: Could not connect to the database. Access denied for user 'SyedMazharAli'@'localhost' (using password: NO) {"name":"SequelizeAccessDeniedError","original":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: NO)","sqlState":"28000"},"parent":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: NO)","sqlState":"28000"},"stack":"SequelizeAccessDeniedError: Access denied for user 'SyedMazharAli'@'localhost' (using password: NO)\n    at ConnectionManager.connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\mysql\\connection-manager.js:118:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at ConnectionManager._connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:332:24)\n    at C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:250:32","timestamp":"2023-04-19T16:13:24.769Z"}
info: Server started at http://localhost:3005/api/v1/ {"timestamp":"2023-04-19T16:13:32.927Z"}
error: Could not connect to the database. Access denied for user 'SyedMazharAli'@'localhost' (using password: NO) {"name":"SequelizeAccessDeniedError","original":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: NO)","sqlState":"28000"},"parent":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: NO)","sqlState":"28000"},"stack":"SequelizeAccessDeniedError: Access denied for user 'SyedMazharAli'@'localhost' (using password: NO)\n    at ConnectionManager.connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\mysql\\connection-manager.js:118:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at ConnectionManager._connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:332:24)\n    at C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:250:32","timestamp":"2023-04-19T16:13:32.992Z"}
info: Server started at http://localhost:3005/api/v1/ {"timestamp":"2023-04-19T16:14:47.006Z"}
error: Could not connect to the database. connect ETIMEDOUT {"name":"SequelizeConnectionError","original":{"code":"ETIMEDOUT","errorno":"ETIMEDOUT","fatal":true,"syscall":"connect"},"parent":{"code":"ETIMEDOUT","errorno":"ETIMEDOUT","fatal":true,"syscall":"connect"},"stack":"SequelizeConnectionError: connect ETIMEDOUT\n    at ConnectionManager.connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\mysql\\connection-manager.js:126:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at ConnectionManager._connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:332:24)\n    at C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:250:32","timestamp":"2023-04-19T16:14:57.024Z"}
info: Server started at http://localhost:3005/api/v1/ {"timestamp":"2023-04-19T16:15:16.681Z"}
error: Could not connect to the database. connect ETIMEDOUT {"name":"SequelizeConnectionError","original":{"code":"ETIMEDOUT","errorno":"ETIMEDOUT","fatal":true,"syscall":"connect"},"parent":{"code":"ETIMEDOUT","errorno":"ETIMEDOUT","fatal":true,"syscall":"connect"},"stack":"SequelizeConnectionError: connect ETIMEDOUT\n    at ConnectionManager.connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\mysql\\connection-manager.js:126:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at ConnectionManager._connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:332:24)\n    at C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:250:32","timestamp":"2023-04-19T16:15:26.698Z"}
info: Server started at http://localhost:3005/api/v1/ {"timestamp":"2023-04-19T16:15:45.303Z"}
error: Could not connect to the database. Access denied for user 'SyedMazharAli'@'localhost' (using password: NO) {"name":"SequelizeAccessDeniedError","original":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: NO)","sqlState":"28000"},"parent":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: NO)","sqlState":"28000"},"stack":"SequelizeAccessDeniedError: Access denied for user 'SyedMazharAli'@'localhost' (using password: NO)\n    at ConnectionManager.connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\mysql\\connection-manager.js:118:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at ConnectionManager._connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:332:24)\n    at C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:250:32","timestamp":"2023-04-19T16:15:45.366Z"}
info: Server started at http://localhost:3005/api/v1/ {"timestamp":"2023-04-19T16:21:35.259Z"}
error: Could not connect to the database. Access denied for user 'SyedMazharAli'@'localhost' (using password: NO) {"name":"SequelizeAccessDeniedError","original":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: NO)","sqlState":"28000"},"parent":{"code":"ER_ACCESS_DENIED_ERROR","errno":1045,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' (using password: NO)","sqlState":"28000"},"stack":"SequelizeAccessDeniedError: Access denied for user 'SyedMazharAli'@'localhost' (using password: NO)\n    at ConnectionManager.connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\mysql\\connection-manager.js:118:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at ConnectionManager._connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:332:24)\n    at C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:250:32","timestamp":"2023-04-19T16:21:35.327Z"}
info: Server started at http://localhost:3005/api/v1/ {"timestamp":"2023-04-19T16:33:34.744Z"}
error: Could not connect to the database. Access denied for user 'SyedMazharAli'@'localhost' to database 'express' {"name":"SequelizeConnectionError","original":{"code":"ER_DBACCESS_DENIED_ERROR","errno":1044,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' to database 'express'","sqlState":"42000"},"parent":{"code":"ER_DBACCESS_DENIED_ERROR","errno":1044,"sqlMessage":"Access denied for user 'SyedMazharAli'@'localhost' to database 'express'","sqlState":"42000"},"stack":"SequelizeConnectionError: Access denied for user 'SyedMazharAli'@'localhost' to database 'express'\n    at ConnectionManager.connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\mysql\\connection-manager.js:126:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at ConnectionManager._connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:332:24)\n    at C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:250:32","timestamp":"2023-04-19T16:33:34.800Z"}
info: Server started at http://localhost:3005/api/v1/ {"timestamp":"2023-04-19T16:36:31.712Z"}
error: Could not connect to the database. Unknown database 'express' {"name":"SequelizeConnectionError","original":{"code":"ER_BAD_DB_ERROR","errno":1049,"sqlMessage":"Unknown database 'express'","sqlState":"42000"},"parent":{"code":"ER_BAD_DB_ERROR","errno":1049,"sqlMessage":"Unknown database 'express'","sqlState":"42000"},"stack":"SequelizeConnectionError: Unknown database 'express'\n    at ConnectionManager.connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\mysql\\connection-manager.js:126:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at ConnectionManager._connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:332:24)\n    at C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:250:32","timestamp":"2023-04-19T16:36:31.772Z"}
info: Server started at http://localhost:3005/api/v1/ {"timestamp":"2023-04-19T16:37:01.617Z"}
error: Could not connect to the database. Unknown database 'express' {"name":"SequelizeConnectionError","original":{"code":"ER_BAD_DB_ERROR","errno":1049,"sqlMessage":"Unknown database 'express'","sqlState":"42000"},"parent":{"code":"ER_BAD_DB_ERROR","errno":1049,"sqlMessage":"Unknown database 'express'","sqlState":"42000"},"stack":"SequelizeConnectionError: Unknown database 'express'\n    at ConnectionManager.connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\mysql\\connection-manager.js:126:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at ConnectionManager._connect (C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:332:24)\n    at C:\\Users\\SyedMazharAli\\Desktop\\postgre\\project\\node_modules\\sequelize\\src\\dialects\\abstract\\connection-manager.js:250:32","timestamp":"2023-04-19T16:37:01.680Z"}
info: Server started at http://localhost:3005/api/v1/ {"timestamp":"2023-04-19T16:38:37.476Z"}
info: Connected to the database. {"timestamp":"2023-04-19T16:38:37.536Z"}
